Core/Aspects/Logging/BaseLogAspect.cs

```diff
@@ -13,7 +13,7 @@ public BaseLogAspect(Type loggerService)
     {
         if (loggerService.BaseType != typeof(LoggerServiceBase))
         {
-            throw new Exception("Wrong validation type!");
+            throw new System.Exception("Wrong validation type!");
```

Comment:
I advise you to always add namespaces using in the beginning of file to reduce extra type length in code